blivet/devices/lvm.py

```diff
@@ -1786,8 +1786,9 @@ def __init__(self, cached_lv, size=None, md_size=None, exists=False, fast_pvs=No
 
     @property
     def size(self):
-        if self.exists:
-            return self.stats.size
+        stats = self.stats
```

Comment:
Because the ``stats`` property is not a static value, we always fetch the fresh data from LVM. It is a cheap operation since it is implemented in libblockdev by looking directly at device-mapper's data via libdevmapper, but there's no reason to do it twice in a row here.